design/oss_suite_release.md

```diff
@@ -0,0 +1,300 @@
+    to determine the state of the automated tests and whether additional integration
+    tests need to be added?
+  - When a new suite release is being prepared, is there a way to run the automated
+    integration tests in each of the components such that the specific versions
+    that will be included in the release will be tested together? If so, is there
+    a way to visualize the status of this set of builds?
+- What updates do we want to make to the official documentation to ensure end users
+  are aware of the OSS suite releases?
+- Assuming we follow semantic versioning as usual for the OSS suite release, what does
+  a major version bump mean?
+  - Proposal: Since the core of this suite is the Conjur server, we would not bump
+    the major version of the suite unless the major version of Conjur changes.
+  - When other components are updated with breaking changes (eg a complete redesign),
+    rather than change the major version we may manually update the release to
+    include the new version as well as the old one and announce a deprecation timeline
+    for the old version before it is removed from the release entirely.
+- Are all of the Summon providers part of the Conjur OSS Suite Release, or just the
+  Summon Conjur Provider? If not, should we have a separate Summon Suite release?
+  Should we consider moving to Summon2 (the version of Summon built into Secretless,
+  that includes all providers)?
+
+### Out of Scope
+- Including any other public repositories beyond those listed above in the initial
+  Conjur OSS Suite Release or on the GitHub landing page.
+- Creating a landing page with any functionality beyond a curated list of Conjur
+  OSS Suite repositories, with descriptions and links.
+- Implementing more than the framework for end-to-end tests and a very basic
+  initial end-to-end test for the Conjur OSS Suite. For more comprehensive test
+  coverage, we will continue to rely on the integration tests written in the
+  individual components, and in future efforts we will review their
+  comprehensiveness and determine potential improvements / ways to leverage the
+  central Conjur OSS Suite Release repo.
+- Automatically including deprecated versions of components during temporary
+  deprecation windows.
+- Plans for addressing security vulnerabilities in collaboration with downstream
+  projects.
+
+### Future Work
+- Augmenting landing page to have an easy way to submit a GitHub issue and be
+  directed to the right repo's backlog.
+- Creating a template for the user-facing READMEs of the component repos beyond
+  just directing them to the OSS suite (eg what basic info should always be included
+  in the GitHub READMEs, which by custom in open source contains basic usage info).
+- Published contributor guidelines are available in a central place in GitHub.
+
+## User Stories
+
+### Open Source Users
+
+|||
+|---|---|
+|AS A|Conjur Open Source user|
```

Comment:
I think that the ALL CAPS for template words is a little too screamy.  Maybe use italics?